src/mantine-core/src/NumberInput/NumberInput.test.tsx

```diff
@@ -339,20 +341,12 @@ describe('@mantine/core/NumberInput', () => {
     expectValue('6');
   });
 
-  it('sets state to min if input is empty and is incremented/decremented', async () => {
+  it('sets state to min if input is empty and is incremented', async () => {
```

Comment:
I noticed that when the input is empty, the decrement button is disabled. However, initially, it was still to possible to interact with it, despite it being disabled.

Since after modifying the disabled style of `NumberInput`, it is no longer possible to click on the buttons. 
I've erased the section of this test that touches of the decrement button.